src/base.config.ts

```diff
@@ -190,7 +174,12 @@ export default function webpackConfigFactory(args: any): webpack.Configuration {
 						getUMDCompatLoader({ bundles: args.bundles }),
 						{
 							loader: 'ts-loader',
-							options: { onlyCompileBundledFiles: true, instance: jsonpIdent, transpileOnly: true, compilerOptions }
+							options: {
+								onlyCompileBundledFiles: false,
+								instance: jsonpIdent,
+								transpileOnly: false,
```

Comment:
you can just remove this flag (it defaults to true).

Reply:
Done!